src/OpenSilver.TemplateWizards/AppCustomizationWindow/PlatformsView.xaml

```diff
@@ -0,0 +1,194 @@
+                                    <VisualState x:Name="Unselected"/>
+                                    <VisualState x:Name="Selected">
+                                        <Storyboard>
+                                            <DoubleAnimation Storyboard.TargetName="selectionIndicator"
+                                                             Storyboard.TargetProperty="Opacity"
+                                                             To=".75"
+                                                             Duration="0"/>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="title" Storyboard.TargetProperty="Foreground">
+                                                <DiscreteObjectKeyFrame KeyTime="0">
+                                                    <DiscreteObjectKeyFrame.Value>
+                                                        <SolidColorBrush Color="#FFFFFF"/>
+                                                    </DiscreteObjectKeyFrame.Value>
+                                                </DiscreteObjectKeyFrame>
+                                            </ObjectAnimationUsingKeyFrames>
+                                            <ObjectAnimationUsingKeyFrames Storyboard.TargetName="selectedBorder"
+                                                                           Storyboard.TargetProperty="Visibility"
+                                                                           Duration="0">
+                                                <DiscreteObjectKeyFrame KeyTime="0">
+                                                    <DiscreteObjectKeyFrame.Value>
+                                                        <Visibility>Visible</Visibility>
+                                                    </DiscreteObjectKeyFrame.Value>
+                                                </DiscreteObjectKeyFrame>
+                                            </ObjectAnimationUsingKeyFrames>
+                                        </Storyboard>
+                                    </VisualState>
+                                </VisualStateGroup>
+                                <VisualStateGroup x:Name="FocusStates">
+                                    <VisualState x:Name="Focused"/>
+                                    <VisualState x:Name="Unfocused"/>
+                                </VisualStateGroup>
+                            </VisualStateManager.VisualStateGroups>
+
+                            <Border x:Name="mouseOverIndicator"
+                                    Background="White"
+                                    BorderBrush="Transparent"
+                                    BorderThickness="{TemplateBinding BorderThickness}"
+                                    CornerRadius="10"
+                                    IsHitTestVisible="False"
+                                    Opacity="0"/>
+                            <Grid>
```

Comment:
It makes it possible to select a platform when clicking between the icon and its title
```suggestion
                            <Grid Background="Transparent">
```